specs/agents/logging.md

```diff
@@ -43,7 +43,7 @@ Also, the documentation should not mention the old log levels going forward.
 
 Agents may provide the following log-related features:
 
-- [Log correlation](log-correlation.md): inject current trace/transaction/error ID into logs.
+- [Log correlation](log-correlation.md): inject current service/trace/transaction/error ID into logs.
```

Comment:
I'd more clearly differentiate the id-based log correlation from adding service metadata. Service metadata isn't really injected into logs. It's a global configuration in the ECS logging libraries, rather than per-event.
Also describe the different features log correlation (logs tab in waterfall) and service metadata (logs tab in service overview) enable.

Reply:
I have made a clear distinction in https://github.com/elastic/apm/pull/766/commits/21597350ad640d4ccce653c446a386c3735333d7 let me know of that works better for you.